serverless.yaml

```diff
@@ -10,8 +10,7 @@ custom:
   resourceTableName: 'resource-db-${self:custom.stage}'
   stage: ${opt:stage, self:provider.stage}
   region: ${opt:region, self:provider.region}
-  oauthCallback: ${opt:oauthCallback, 'http://localhost'}
-  oauthRedirect: ${opt:oauthRedirect, 'http://localhost'}
+  oauthDomain: ${opt:oauthCallback, 'https://dev-6460611.okta.com/oauth2/default'}

```

Comment:
should there be a specific okta reference here?

Reply:
Good callout this should be a less precise default